CHANGELOG.ios.md

```diff
@@ -4,6 +4,17 @@
 - [iOS 2.0 SDK](http://aka.ms/gc6fex)
```

Comment:
Yeah, I was planning of updating this to 2.2 when I update the link (after testing the release build)